packages/calypso-e2e/src/media-helper.ts

```diff
@@ -26,12 +32,32 @@ export function getScreenshotDir(): string {
  * @returns {string} Absolute path to the directory.
  */
 export function getVideoDir(): string {
-	return path.resolve(
-		process.env.TEMP_ASSET_PATH || path.join( __dirname, '..' ),
-		process.env.VIDEODIR || 'screenshots/videos'
-	);
+	return path.resolve( getAssetDir(), process.env.VIDEODIR || 'screenshots/videos' );
 }
 
+/**
+ * Returns a descriptive file name for the screenshot file.
+ *
+ * @param {string} name Name of the test case that failed.
+ * @param {string} state Pass/fail state of the test case.
+ * @returns {string} A Path-like string.
+ */
+export function getScreenshotName( name: string, state: string ): string {
+	const shortTestFileName = name.replace( /[^a-z0-9]/gi, '-' ).toLowerCase();
+	const screenSize = getTargetScreenSize().toUpperCase();
+	const locale = getTargetLocale().toUpperCase();
+	const date = getDateString();
+	const fileName = `${ state }-${ locale }-${ screenSize }-${ shortTestFileName }-${ date }`;
```

Comment:
Do we actually need the `state` info? Is there a scenario where we screenshot a passed test?

Reply:
I was trying to mirror the Selenium behavior of `saveScreenshot` where there is provision to specify that even passing screenshots be saved.

Sergio found in #52617 that those options were not being used anywhere though, so he has deprecated it. I think we can do the same here.